docs/docs/monitoring/analytics/data-structure-reference.mdx

```diff
@@ -781,6 +781,120 @@ Time of the last update of the slot in this session. The timestamp is a UTC.
 - Type: `DateTime`
 - Example: `2022-06-21 02:15:49.326936`
 
+## rasa_patterns
+
+Patterns are marker definitions that have been received from Rasa. This table is called `patterns`
+to distinguish them from extracted markers which are stored in `rasa_markers` table. It 
+stores the configuration of markers (which can be thought of as pattern
```

Comment:
as **a** pattern of conversational events :)

Reply:
Oh right, I'll fix it

Reply:
Fixed, thanks!